app/models/updateVatSubscription/request/UpdatedOrganisationDetails.scala

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.updateVatSubscription.request
+
+import play.api.libs.json.{Json, Writes}
+
+case class UpdatedOrganisationDetails(organisationName: Option[String],
+                                      individualName: Option[IndividualName],
+                                      tradingName: Option[String])
+
+object UpdatedOrganisationDetails {
+
+  implicit val writes: Writes[UpdatedOrganisationDetails] = Writes {
+    model => Json.obj(
```

Comment:
When I try to test this manually with Postman, we're getting an error back from our stub to say that the update request does not validate against the schema. I can see the JSON that is being sent to the stub and when any of these fields are empty it's sending them as `null`, e.g. `"organisationName":null`. That's what is causing the schema validation error. Can you use the `jsonObjNoNulls` function in our helper trait `JsonObjectSugar` here to remove the null values so that empty fields are not written at all.